README.md

```diff
@@ -152,7 +153,7 @@ in `config.yaml`). You may also provide a CIDR block.
 
 You can do this with your own tool (or SQL) Archon comes with a small utility for adding accounts:
 
-    ./add_account
+    ./account --add
```

Comment:
I didn't realize these were flags, can you make this more like a conventional *nix-y subcommand? I.e `account [add|delete|...] ...`